src/Components/Slider/Slider.js

```diff
@@ -1,15 +1,72 @@
 import React, { Component } from "react";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
 import "./Slider.scss";
+import Slider from "react-slick";
```

Comment:
라이브러리 import 문을 위로 올려주세요~
```suggestion
import React, { Component } from "react";
import Slider from "react-slick";
import "slick-carousel/slick/slick.css";
import "slick-carousel/slick/slick-theme.css";
import "./Slider.scss";
```